spec/integration/plugins/associates_spec.rb

```diff
@@ -58,33 +59,98 @@
     end
 
     context 'with a schema' do
-      include_context 'automatic FK setting', schema: true do
+      include_context 'automatic FK setting'
+
+      before do
+        configuration.relation_classes[1].class_eval do
+          schema do
+            attribute :id, ROM::SQL::Types::Serial
+            attribute :user_id, ROM::SQL::Types::ForeignKey(:users)
+            attribute :title, ROM::SQL::Types::String
+
+            associate do
+              belongs :users, as: :user
```

Comment:
This seems harmless, I think. In general we avoid implicit behaviors (aka magic) but in this case it seems like a pragmatic choice as this association will be commonly used for aggregates where you definitely want a singular name so...